packages/esm-patient-medications-app/src/order-basket/order-basket-search-results.component.tsx

```diff
@@ -47,62 +49,60 @@ export default function OrderBasketSearchResults({
           </div>
-
           {currentSearchResultPage.map((result, index) => (
             <ClickableTile
+              light={isTablet}
+              role="listitem"
               key={index}
-              className={styles.searchResultTile}
-              style={{ marginTop: '5px' }}
+              className={isTablet ? `${styles.tabletSearchResultTile}` : `${styles.desktopSearchResultTile}`}
               handleClick={() => handleSearchResultClicked(result, false)}>
-              <div
-                style={{
-                  display: 'flex',
-                  justifyContent: 'space-between',
-                  alignItems: 'center',
-                }}>
-                <Medication16 style={{ flex: '0 0 auto', marginRight: '20px' }} />
-                <div style={{ flex: '1 1 auto' }}>
+              <div className={styles.searchResultTile}>
+                <Medication16 className={styles.medicationIcon} />
+                <div className={styles.searchResultTileContent}>
                   <p>
-                    <strong>
-                      {result.drug.concept.display} ({result.dosage?.dosage})
-                    </strong>
+                    <strong>{result.drug.concept.display}</strong> &mdash; {result.dosage?.dosage} &mdash;{' '}
+                    {result.dosageUnit.name}
                     <br />
-                    <span className={styles.label01}>{result.dosageUnit.name}</span> &mdash;{' '}
                     <span className={styles.label01}>{result.frequency.name}</span> &mdash;{' '}
                     <span className={styles.label01}>{result.route.name}</span>
                   </p>
                 </div>
                 <Button
-                  style={{ flex: '0 0 auto' }}
+                  className={styles.addToBasketButton}
                   kind="ghost"
                   hasIconOnly={true}
-                  renderIcon={() => <ShoppingBag16 />}
+                  renderIcon={() => <ShoppingCart16 />}
                   iconDescription={t('directlyAddToBasket', 'Immediately add to basket')}
                   onClick={() => handleSearchResultClicked(result, true)}
                 />
               </div>
             </ClickableTile>
           ))}
           {searchResults.length > 0 && (
-            <Pagination
-              page={page}
-              pageSize={pageSize}
-              pageSizes={[10, 20, 30, 40, 50]}
-              totalItems={searchResults.length}
-              onChange={({ page, pageSize }) => {
-                setPage(page);
-                setPageSize(pageSize);
-              }}
-            />
+            <>
+              <Pagination
+                page={page}
+                pageSize={pageSize}
+                pageSizes={[10, 20, 30, 40, 50]}
+                totalItems={searchResults.length}
+                onChange={({ page, pageSize }) => {
+                  setPage(page);
```

Comment:
Is there a reason to set both values? Shouldn't we only set the value that changed?

Reply:
Dunno why the original author went for this implementation. I see this approach is being used in multiple patient chart widgets. That said, Ciaran actually wants this pagination removed altogether (I can file a ticket for that).

<img width="481" alt="Screenshot 2021-09-12 at 19 30 12" src="https://user-images.githubusercontent.com/8509731/132995431-aa742ff5-809a-4471-9ab9-81aeb6afe80b.png">

Reply:
Sounds good to me!